examples/basic/.yalc/swingset/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "swingset",
   "description": "drop-in component library and documentation pages for next.js",
-  "version": "0.13.0",
+  "version": "0.14.0",
```

Comment:
`0.13.1` perhaps?

Reply:
Will bump to `0.14.1`, but this is just via `yalc` for local testing!